packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -1089,6 +1089,8 @@ export class API extends EventEmitter {
   // * @param {Boolean} opts.twoStep[=false] - Optional: use 2-step balance computation for improved performance
   // * @param {Boolean} opts.includeExtendedInfo (optional: query extended status)
   // * @param {String} opts.tokenAddress (optional: ERC20 Token Contract Address)
+  // * @param {String} opts.multisigContractAddress (optional: MULTISIG ETH Contract Address)
+  // * @param {String} opts.network (optional: MULTISIG ETH Contract Address network)
```

Comment:
why we need network here? Should the multisig contract use the same network as the original wallet?